modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/EditAPIApplication.testcase

```diff
@@ -29,6 +29,245 @@ definition {
 		}
 	}
 
+	@priority = 4
+	test CanEditSchemaNameAndDescriptionInPublishedAPIApplication {
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("Given create an published API application with related schema") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
```

Comment:
Thanks for the codes, in headless team, we would like to use API to create the api builder in "Given" and "And Given" steps. Please help update to  ApplicationAPI.createAPIApplication() accordingly, thanks.

Reply:
Done! Thanks for this review @yanan